AndroidSDKCore/src/main/java/com/leanplum/internal/SocketIOClient.java

```diff
@@ -78,15 +77,17 @@ private static String userAgentString() {
         + Constants.LEANPLUM_VERSION + "/" + Constants.LEANPLUM_PACKAGE_IDENTIFIER + ")";
   }
 
-  private static String downloadUriAsString(final HttpUriRequest req)
+  private static String downloadUriAsString(final String urlString)
```

Comment:
Looks like the change is to private method. But, i do not see the callers of these method change to the new signature. Can we please make sure that callers are updated and the code runs all test cases successfully?